[Windows] npm install fails on ./bower-lite

I'm trying to get things working on Windows with the latest jupyterhub master and get the below error from `npm install`
```
(jupyterhub) C:\dev\src\jupyterhub>npm install --unsafe-perm

> jupyterhub-deps@0.0.0 postinstall C:\dev\src\jupyterhub
> ./bower-lite

'.' is not recognized as an internal or external command,
operable program or batch file.

npm WARN jupyterhub-deps@0.0.0 license should be a valid SPDX license expression
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Miniconda3\\envs\\jupyterhub\\node.exe" "C:\\Miniconda3\\envs\\jupyterhub\\node_modules\\npm\\bin\\npm-cli.js" "install" "--unsafe-perm"
npm ERR! node v6.13.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! jupyterhub-deps@0.0.0 postinstall: `./bower-lite`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyterhub-deps@0.0.0 postinstall script './bower-lite'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyterhub-deps package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./bower-lite
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyterhub-deps
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyterhub-deps
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\dev\src\jupyterhub\npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "url": "https://github.com/jupyter/jupyterhub.git"
   },
   "scripts": {
-    "postinstall": "./bower-lite",
+    "postinstall": "python ./bower-lite",
     "lessc": "lessc"
   },
   "devDependencies": {

